Code/Tools/FBuild/FBuildCore/Helpers/VSProjectGenerator.cpp

```diff
@@ -102,26 +146,69 @@ const AString & VSProjectGenerator::GenerateVCXProj( const AString & projectFile
     const char * lastSlash = projectFile.FindLast( NATIVE_SLASH );
     AStackString<> projectBasePath( projectFile.Get(), lastSlash ? lastSlash + 1 : projectFile.Get() );
 
+    const bool writeCSProj		= projectFile.EndsWithI( ".csproj" );
+    const bool writeAndroidProj = projectFile.EndsWithI( ".androidproj" );
+
+    // Don't write a custom build command by default, because as of Visual Studio 2015 R3, Makefile projects can't currently utilize this functionality
+    // However, if we have a Utility project, then we can execute per-file custom build steps
+    bool writeCustomBuildCommands = false;
+
     // header
     Write( "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n" );
-    Write( "<Project DefaultTargets=\"Build\" ToolsVersion=\"4.0\" xmlns=\"http://schemas.microsoft.com/developer/msbuild/2003\">\n" );
+    Write( "<Project DefaultTargets=\"Build\" ToolsVersion=\"14.0\" xmlns=\"http://schemas.microsoft.com/developer/msbuild/2003\">\n" );
```

Comment:
I wonder where this 4.0 came from originally.  That certainly seems wrong.

Reply:
Maybe a typo where the 1 was accidentally deleted?